docs/using.rst

```diff
@@ -325,6 +265,6 @@ Some Useful Commands
 * ``vagrant destroy`` - Destroy the Vagrant Box:
 
   **Warning:**
-  Using ``vagrant destroy``, will destroy any data you have stored in the Vagrant
+  Using ``vagrant destroy`` will destroy any data you have stored in the Vagrant
```

Comment:
remove "have", it's the most ambiguous word in the English language[1] and 99% of the time isn't necessary.

[1] https://en.wikipedia.org/wiki/Perfect_(grammar)